lib/src/kbasesearchengine/main/NarrativeInfoDecorator.java

```diff
@@ -153,9 +182,17 @@ public GetObjectsOutput getObjects(final GetObjectsInput params, final String us
             }
         }
         catch (IOException | Auth2Exception e) {
-            LoggerFactory.getLogger(getClass()).error("ERROR: Failed retrieving workspace owner realname(s): " +
+            getLogger().error("ERROR: Failed retrieving workspace owner realname(s): " +
```

Comment:
I didn't notice this before, but this seems like weird behavior to me - it's returning partial results on a failure, which is really a pain for users of the API. Maybe not in this PR, but in a future PR, it should either retry or throw an error IMO.